packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -920,16 +863,6 @@ void _exitWithUnsupportedProjectMessage() {
   );
 }
 
-void _exitWithProjectNotUsingGradleMessage() {
```

Comment:
This is for projects created around ~2016, I think it is safe to remove now

Reply:
I think that it's not worth keeping a feature around if it's not used. I don't think it's important either, since we neither have unit or integration tests, nor do we have any backlog issues about migrating from Gradle.

Reply:
err "to" gradle, not "from" gradle.